common/workunit/workunit.cpp

```diff
@@ -2881,19 +2881,127 @@ class CDaliWorkUnitFactory : public CWorkUnitFactory, implements IDaliClientShut
                                                 ISecManager *secmgr,
                                                 ISecUser *secuser)
     {
+        class CQueryOrFilter : public CInterface, implements IInterface
+        {
+            int fmt;
+            StringAttr name;
+            StringArray values;
+        public:
+            IMPLEMENT_IINTERFACE;
+
+            CQueryOrFilter() {};
+
+            void setName(const char* _name) { name.set(_name); };
+            void setFormat(int _fmt) { fmt = _fmt; };
+            void appendValue(const char* _value) { values.append(_value); };
+            const char* getName() { return name.get(); };
+            int getFormat() { return fmt; };
+            StringArray& getValues() { return values; };
+        };
+        class CWorkUnitsPagerHelper : public CInterface, implements IInterface
```

Comment:
This seems to be a class that collects global functions - it has no members and implements no sensible interfaces.  I think it only makes the code more confusing.